src/ASM/GlbL2projector.C

```diff
@@ -33,6 +34,102 @@ LinAlg::MatrixType GlbL2::MatrixType   = LinAlg::SPARSE;
 LinSolParams*      GlbL2::SolverParams = nullptr;
 
 
+static void expandTensorGrid2 (const RealArray* in, RealArray* out)
```

Comment:
Fine. But you may want to add some small doxy for the these static functions? Or else I assume some jenkins check will complain about it.

Reply:
yeah, copied from originals.